libs/docusaurus/src/schematics/app/schematic.ts

```diff
@@ -156,9 +156,9 @@ export default function (options: AppSchematicSchema): Rule {
     updatePrettierIgnore(),
     addDepsToPackageJson(
       {
-        '@docusaurus/core': '^2.0.0-alpha.59',
-        '@docusaurus/preset-classic': '^2.0.0-alpha.59',
-        classnames: '^2.2.6',
+        '@docusaurus/core': '^2.0.0-alpha.61',
+        '@docusaurus/preset-classic': '^2.0.0-alpha.61',
```

Comment:
I'd like to see these versions pinned, but we shouldn't do that now because the change I made in Docusaurus was undone. I think it was unintentional and the result of running a Lerna command. We'll have to stick with this for now.